File: Java/MenuBancarioSwitch.java
import java.util.Scanner;

public class MenuBancarioSwitch {
  public static void main(String[] args) {

    Scanner scanner = new Scanner(System.in);
    int numero = 0;
    int cantidad = 0;
    int saldo = 1000;

    while (numero != 4) {
      System.out.println("Menú: ");
      System.out.println("1 - Extracto de cuenta: ");
      System.out.println("2 - Ingreso en cuenta: ");
      System.out.println("3 - Reintegro de cuenta: ");
      System.out.println("4 - Salir ");
      System.out.print("Elije Opcion: ");
      numero = scanner.nextInt();

      System.out.println("-------------------");
      switch (numero) {
        case 1:
          System.out.println("Tienes " + saldo + " Euros de Saldo.");
          break;
        case 2:
          System.out.println("¿Que Cantidad quieres Ingresar?");
          cantidad = scanner.nextInt();
          saldo += cantidad;
          System.out.println("Ingresando " + cantidad + " en Cuenta, tienes " + saldo + " Euros");
          break;
        case 3:
          System.out.println("¿Que Cantidad quieres Reintegrar?");
          cantidad = scanner.nextInt();
          saldo -= cantidad;
          System.out.println("Reintegro de " + cantidad + " Euros, tienes " + saldo + " Euros");
          break;
        case 4:
          System.out.println("Hasta Pronto!");
          break;
        default:
          System.out.println("Opcion no Valida!");
          break;
      }
      System.out.println("-------------------");
    }  
    scanner.close();
  }
}

File: C/MenuBancarioSwitch.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
  int saldo = 1000;
  int cantidad = 0;
  int opcion = 0;

  system("clear");

  while ( opcion != 4)
  {
    puts("Esto es un Menu Bancario: ");
    puts("1 - Extracto de cuenta: ");
    puts("2 - Ingresar en cuenta: ");
    puts("3 - Reintegro de cuenta: ");
    puts("4 - Salir ");
    printf("Elije Opcion: ");
    scanf("%d", &opcion);

    puts("-------------------");
    switch (opcion)
    {
    case (1):
      printf("Tienes %d Euros de Saldo.\n", saldo);
      break;
    case (2):
      printf("¿Que cantidad quiere Ingresar?: ");
      scanf("%d", &cantidad);
      saldo += cantidad;
      printf("Ingresando %d en Cuenta, tienes %d Euros\n", cantidad, saldo);
      break;
    case (3):
      printf("¿Que cantidad quiere Retirar?: ");
      scanf("%d", &cantidad);
      saldo -= cantidad;
      printf("Reintegro de %d Euros, tienes %d Euros\n", cantidad, saldo);
      break;
    case (4):
      puts("Hasta Pronto!");
      break;
    default:
      puts("¡Opcion no valida!");
      break;
    }
    puts("-------------------");
  }

  return 0;
}